modules/mod-mp3/ExportMP3.cpp

```diff
@@ -1702,9 +1701,6 @@ bool MP3ExportProcessor::Initialize(AudacityProject& project,
    context.channels = channels;
 
    int rate = lrint(sampleRate);
-#ifndef DISABLE_DYNAMIC_LOADING_LAME
```

Comment:
It is really odd that this code is still here, provided that Audacity links against lame directly for quite a few years. I think that we need to revise ExportMP3 code.

Reply:
That could be separately addressed of course.  What do you think of this PR otherwise?